src/listview_curses.cc

```diff
@@ -136,6 +136,15 @@ bool listview_curses::handle_key(int ch)
         this->shift_top(this->rows_available(this->lv_top, RD_DOWN) - vis_line_t(1));
         break;
 
+
+    case 'u':
```

Comment:
Unfortunately, both of these keys are already used: u - next bookmark; d - move forward by 24 hours (look in hotkeys.cc for the implementations).  Unfortunately, there's just not that many keys that aren't in use at this point.  I'm not sure what to do about it.

Reply:
(Apologies for replying on such an old thread.)

It's a bit sad having more less-like shortcuts is not possible - like `f` for scrolling forward (I have never used space to move one page down, and it's now embedded in muscle memory), or `d` and `u` for half page navigation since it's extremely useful.

Given that I don't use bookmarks and I never use by-time navigation, it seems a bit weird that otherwise useful keyboard shortcuts are sacrificed to those functions, at least in my particular case...